tests/loggers/test_all.py

```diff
@@ -96,3 +96,28 @@ def test_loggers_pickle(tmpdir, monkeypatch, logger_class):
 
     trainer2 = pickle.loads(pkl_bytes)
     trainer2.logger.log_metrics({'acc': 1.0})
+
+
+@pytest.mark.parametrize("extra_params", [
+    pytest.param(dict(max_epochs=1, auto_scale_batch_size=True), id='Batch-size-Finder'),
+    pytest.param(dict(max_epochs=10, auto_lr_find=True), id='LR-Finder'),
+])
+def test_logger_reset_correctly(tmpdir, extra_params):
+    """ Test that logger is updated correctly """
```

Comment:
```suggestion
    """ Test that the tuners do not alter the logger reference """
```
Looking at the test, it seems like it should not update.